src/lib/server/websearch/generateQuery.ts

```diff
@@ -1,19 +1,66 @@
 import type { Message } from "$lib/types/Message";
 import { format } from "date-fns";
 import { generateFromDefaultEndpoint } from "../generateFromDefaultEndpoint";
-import { defaultModel } from "../models";
+import { smallModel } from "../models";
 
 export async function generateQuery(messages: Message[]) {
 	const currentDate = format(new Date(), "MMMM d, yyyy");
 	const userMessages = messages.filter(({ from }) => from === "user");
 	const previousUserMessages = userMessages.slice(0, -1);
+
 	const lastMessage = userMessages.slice(-1)[0];
-	const promptSearchQuery = defaultModel.webSearchQueryPromptRender({
-		message: lastMessage,
-		previousMessages: previousUserMessages.map(({ content }) => content).join(" "),
-		currentDate,
+
+	const convQuery: Array<Omit<Message, "id">> = [
+		{
+			from: "user",
+			content: `Previous Questions:
+- Who is the president of France?
+
+Current Question: What about Mexico?
+`,
+		},
+		{
+			from: "assistant",
+			content: 'query: "President of Mexico"',
```

Comment:
Why do you need query: here? this looks cleaner:

```suggestion
			content: "President of Mexico",
```

Reply:
Ok I would probably refactor this too but can be later.

Reply:
I think with the examples we could tell the model to answer with the query directly and then we don't need the regex, just use the output directly as the web search query. I'll try if it doesn't affect performance i'll push it in this pr

Reply:
works ok! pushed the changes

Reply:
nice! Way cleaner